src/installation/raspi.adoc

```diff
@@ -9,7 +9,7 @@ endif::[]
 == Introduction
 
 This guide is for compiling FORTE for the Raspberry Pi. 
-For information about the parameters to be used, go to link:../../html/parameters/parameters.html#sysfs[Parameters].
+For information about the parameters to be used, go to xref:../../html/io_config/SysFs.adoc[Parameters].
```

Comment:
I tried this new link, unfortunately it seems that is does not work. Is the full link https://github.com/eclipse-4diac/4diac-documentation/blob/main/src/io_config/SysFs.adoc? Then I think the link sould be ../io_config/SysFs.adoc right?

Reply:
ah, damn, fixed now.